test/inductor/test_op_completeness.py

```diff
@@ -0,0 +1,42 @@
+# Owner(s): ["module: inductor"]
+import unittest
+
+from torch._inductor.codegen.cpp import CppOverrides, CppVecOverrides
+from torch._inductor.codegen.halide import HalideOverrides
+from torch._inductor.codegen.mps import MetalOverrides
+from torch._inductor.codegen.triton import TritonKernelOverrides
+from torch._inductor.ops_handler import list_ops, OP_NAMES
+from torch._inductor.test_case import TestCase
+
+
+class TestOpCompleteness(TestCase):
+    def verify_ops_handler_completeness(self, handler):
+        op_names = list_ops(handler)
+        if OP_NAMES == op_names:
+            return
+        print(f"Missing ops: {OP_NAMES - op_names}")
+        print(f"Extra ops: {op_names - OP_NAMES}")
+        self.assertEqual(", ".join(OP_NAMES - op_names), "")
+        self.assertEqual(", ".join(op_names - OP_NAMES), "")
```

Comment:
I'm not sure it's a good contract to let specific OpsHandler implementation for triton, halide, cpp etc have the exactly the same ops defined as the OpsHandler 'interface' (not exactly an interface but it works like one). Backend may want to have backend specific ops.

Reply:
You can add backend-specific ops by making them start with `_` (which list_ops ignores).